recipes/pept/meta.yaml

```diff
@@ -0,0 +1,84 @@
+# File   : meta.yaml
+# License: GNU v3.0
+# Author : Andrei Leonard Nicusan <a.l.nicusan@bham.ac.uk>
+# Date   : 30.06.2020
+
+
```

Comment:
All feedstocks (i.e. this file) in conda-forge are `BSD-3-Clause`, is that okay?
```suggestion
```